src/app/frontend/common/dialogs/previewdeployment/dialog.ts

```diff
@@ -43,27 +45,20 @@ export class PreviewDeploymentDialog implements OnInit {
     const configuration = this.toDeploymentConfiguration(this.data.spec);
     this.text = JSON.stringify(this.removeEmptyField(configuration));
 
-    this.buttonToggleGroup.valueChange.subscribe((selectedMode: EditorMode) => {
+    this.buttonToggleGroup.valueChange.pipe(takeUntil(this.unsubscribe_)).subscribe((selectedMode: EditorMode) => {
```

Comment:
Most important part.

Reply:
It seems that `src/app/frontend/common/dialogs/editresource/dialog.ts` and `src/app/frontend/crd/crdobject/component.ts` have the same issue.

Reply:
Done.